coops-ui/src/assets/styles/base.styl

```diff
@@ -212,3 +212,18 @@ hr
   color $gray6
   font-size 0.875rem
   font-weight 300
+
+// Network Error
+.network-error
+  text-align center
+
+.network-error__title
+  font-size 1rem
+  font-weight 700
+  color $BCgovInputError
+
+.network-error__subtitle
+  margin-top 0.25rem
+  color $gray6
+  font-size 0.875rem
+  font-weight 300
```

Comment:
I would expect to see these in the Dashboard component, rather than base.styl - that's been our convention when styles only apply to a certain component.